api/spec/requests/groups_controller_spec.rb

```diff
@@ -363,4 +363,33 @@
       end
     end
   end
+
+  # Members
+  describe 'GET /groups/:id/members' do
+    let(:creator) { create(:user) }
+    let(:group) { create(:group, :with_members) }
+    let(:headers) { { 'Authorization' => response.headers['Authorization'] } }
+
+    before do
+      post user_session_path,
+           params: {
+             user: {
+               email: creator.email,
+               password: creator.password
+             }
+           }
+      get members_group_path(group.id), headers:
+    end
+
+    it 'returns a successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'returns JSON containing the creator as the only member' do
+      json_response = response.parsed_body
+      expect(json_response.size).to eq(2)
+      expect(json_response[0]['id']).to be_a(Integer)
+      expect(json_response[0]['email']).to be_a(String)
+    end
```

Comment:
`returns JSON containing the creator as the only member` -> esto no está pasando en el contexto del test, no?
Porque el `creator` nunca se setea al grupo..

Ahora mismo el endpoint está retornando los dos members (`admins` ambos), que se crean con el trait que agregaste:
```ruby
trait :with_members do
  members { build_list :member, 2 }
end
```

<img width="803" alt="image" src="https://github.com/wyeworks/finder/assets/62676004/0ccfde8d-89e4-43b6-bbe0-3537c5d5f239">

Yo capaz lo que haría es:
- Cambiar el trait para que cree los users como participants.
- Agregar el `creator` al grupo que creaste en el test.
- Chequear cuando corre el test que retorne los tres members que el grupo debería tener.

Reply:
Perfecto Juan, toda la razon, lo modifque ahi.